controllers/cdapmaster/cdapmaster_controller.go

```diff
@@ -69,6 +69,9 @@ type UserInterface struct{}
 // SupportBundle - interface to handle cdapmaster
 type SupportBundle struct{}
 
+// SystemMetricsExporter - interface to handle cdapmaster
+type SystemMetricsExporter struct{}
```

Comment:
Do we need to add new services here? My understanding was that this was intended to support CDF installations with old operator.

Reply:
Reverted the commit for this.